RallyHereIntegration/Config/BaseRallyHereIntegration.ini

```diff
@@ -110,4 +124,4 @@ BurstCountThreshold=5
 BurstTimeThresholdInSeconds=5
 
 [/Script/RallyHereDebugTool.RallyHereDebugToolSettings]
-DefaultWindowPositions="[Window][Output Log]\nDockId=0x00000003,0\n[Window][Login]\nDockId=0x00000004,1\n[Window][Session]\nDockId=0x00000005,10\n[Window][Friends]\nDockId=0x00000005,3\n[Window][Config]\nDockId=0x00000005,1\n[Window][Presence]\nDockId=0x00000005,7\n[Window][Web Requests]\nDockId=0x00000003,1\n[Window][Analytics]\nDockId=0x00000003,2\n[Window][About]\nDockId=0x00000003,3\n[Window][Catalog]\nDockId=0x00000005,0\n[Window][Purge]\nDockId=0x00000005,8\n[Window][Entitlements]\nDockId=0x00000005,2\n[Window][Notifications]\nDockId=0x00000005,5\n[Window][Player Repository]\nDockId=0x00000004,0\n[Window][Platforms]\nDockId=0x00000005,6\n[Window][Inventory]\nDockId=0x00000005,4\n[Window][Ranks]\nDockId=0x00000005,9\n[Window][Settings]\nDockId=0x00000005,11\n[Window][Custom Endpoint]\nDockId=0x00000005,12\n[Docking][Data]\nDockNode      ID=0x00000001 Pos=0,19 Size=1633,905 Split=Y\nDockNode    ID=0x00000002 Parent=0x00000001 SizeRef=1008,561 Split=X Selected=0x380E6B3B\nDockNode  ID=0x00000004 Parent=0x00000002 SizeRef=508,300 Selected=0x48510B82\nDockNode  ID=0x00000005 Parent=0x00000002 SizeRef=1123,300 Selected=0x9D227BB7\nDockNode    ID=0x00000003 Parent=0x00000001 SizeRef=1008,337 Selected=0x05E372E7"
\ No newline at end of file
+DefaultWindowPositions="[Window][Output Log]\nDockId=0x00000003,0\n[Window][Login]\nDockId=0x00000004,1\n[Window][Session]\nDockId=0x00000005,10\n[Window][Friends]\nDockId=0x00000005,3\n[Window][Config]\nDockId=0x00000005,1\n[Window][Presence]\nDockId=0x00000005,7\n[Window][Web Requests]\nDockId=0x00000003,1\n[Window][Analytics]\nDockId=0x00000003,2\n[Window][About]\nDockId=0x00000003,3\n[Window][Catalog]\nDockId=0x00000005,0\n[Window][Purge]\nDockId=0x00000005,8\n[Window][Entitlements]\nDockId=0x00000005,2\n[Window][Notifications]\nDockId=0x00000005,5\n[Window][Player Repository]\nDockId=0x00000004,0\n[Window][Platforms]\nDockId=0x00000005,6\n[Window][Inventory]\nDockId=0x00000005,4\n[Window][Ranks]\nDockId=0x00000005,9\n[Window][Settings]\nDockId=0x00000005,11\n[Docking][Data]\nDockNode      ID=0x00000001 Pos=0,19 Size=1633,905 Split=Y\nDockNode    ID=0x00000002 Parent=0x00000001 SizeRef=1008,561 Split=X Selected=0x380E6B3B\nDockNode  ID=0x00000004 Parent=0x00000002 SizeRef=508,300 Selected=0x48510B82\nDockNode  ID=0x00000005 Parent=0x00000002 SizeRef=1123,300 Selected=0x9D227BB7\nDockNode    ID=0x00000003 Parent=0x00000001 SizeRef=1008,337 Selected=0x05E372E7"
```

Comment:
Was this intended to change?

Reply:
No, probably missed an update from Custom Endpoint window being added when I moved it to Git.